pkg/audit/containment_test.go

```diff
@@ -32,6 +32,7 @@ func TestContainIncrementAndGet(t *testing.T) {
 			ShareSize:         0,
 			ExpectedShareHash: pkcrypto.SHA256Hash(testrand.Bytes(10)),
 			ReverifyCount:     0,
+			Path:              "",
```

Comment:
I wonder if it really makes sense to explicitly initialize Path with empty value here. This will be done implicitly anyway. Same is valid of `PieceID`, `StripeIndex`, `ShareSize` and `ReverifyCount`.